src/styles/_variables.scss

```diff
@@ -5,7 +5,7 @@ $schema: $light-material-schema;
 $typeface: $material-typeface;
 $type-scale: $material-type-scale;
 
-$direction: ltr;
```

Comment:
While convenient for testing, I don't think we should change the direction for all samples.